floyd/client/data.py

```diff
@@ -72,10 +74,19 @@ def create(self, data):
             return response.json().get("id")
 
     def get(self, id):
-        response = self.request("GET",
-                                "{}{}".format(self.url, id))
-        data_dict = response.json()
-        return Data.from_dict(data_dict)
+        try:
+            response = self.request("GET",
+                                    "{}{}".format(self.url, id))
+            data_dict = response.json()
+            return Data.from_dict(data_dict)
+        except ClickException as e:
+            floyd_logger.info("Data {}: {}".format(id, e.message))
+            return None
+        except HTTPError as e:
```

Comment:
I'm not following...
`HTTPError` is what's raised by `raise_for_status()` ([see docs here](http://docs.python-requests.org/en/master/api/#requests.Response.raise_for_status)). The base client raises it [here](https://github.com/floydhub/floyd-cli/blob/master/floyd/client/base.py#L81). Am I missing something?